### PR TITLE
Homework nr 6

### PR DESCRIPTION
**Pull request by ArtiomLevitchii**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_5/Weekday.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    THURSDAY("WeekDay"),
+    FRIDAY("WeekDay"),
+    SUNDAY("DayOff"),
+    SATURDAY("DayOff");
+    private String index;
+    Weekday(String index){
+        this.index =index;
+    }
+
+    boolean isWeekDay(){
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java]: Interesting solution :) but it works.

Comment (gh3rtzz) [reply src/main/java/org/example/homework_nr_5/Weekday.java]: maybe?

`boolean isHoliday(){
        if(this == SATURDAY || this == SUNDAY) return true;
    }`

Comment (ArtiomLevitchii) [reply src/main/java/org/example/homework_nr_5/Weekday.java]: It's my fault. I've dove this homework already, and fixed it. I'ts homework_nr_5, i've joined two homeworks, without refreshing changes.

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**